code/LINDA/LINDA_fire.dm

```diff
@@ -131,7 +143,7 @@
 				if(!(location.atmos_adjacent_turfs & direction))
 					continue
 				var/turf/simulated/T = get_step(src, direction)
-				if(istype(T) && T.active_hotspot)
+				if(istype(T) && !T.active_hotspot)
```

Comment:
Because it doesn't make sense otherwise. If it spreads to T that ALREADY has an active hotspot....then it doesn't spread at all since it returns immediately at the beginning of hotspot_expose